### DISCUSSION TITLE
What is the specific meaning of relativeweight in the DEM module? If you can answer this question, I will be grateful, thank you

### DISCUSSION THREAD

**Discussion by latticeyu** [DualSPHysics/DualSPHysics#259] [DUALSPHYSICS_V5.2]
![1d6a529dbf2108081b1136bfc99526a](https://github.com/user-attachments/assets/c1bff16e-50f3-4b66-a177-9c99ae23f65f)

**Comment by alexbexe** [comment]
"relativeweight" is the relative value compared to the density of water (rho0)
if you define water density as 1000 kg/m^3, and then you relativeweight=0.5, that means a density of 500 kg/m^3

**Comment by latticeyu** [comment]
thank you very much for your help



---Original---
From: "Alejandro J. C. ***@***.***&gt;
Date: Fri, Nov 15, 2024 19:54 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DualSPHysics/DualSPHysics] What is the specific meaning ofrelativeweight in the DEM module? If you can answer this question, I will begrateful, thank you (Discussion #259)




 
"relativeweight" is the relative value compared to the density of water (rho0)
 if you define water density as 1000 kg/m^3, and then you relativeweight=0.5, that means a density of 500 kg/m^3
